elfpy/bots/environment_config.py

```diff
@@ -21,14 +20,12 @@ class EnvironmentConfig(types.FrozenClass):
 
     # Logical username for who is running bots
     username: str = DEFAULT_USERNAME
-    # Logical username for who is running bots
-    username_register_url: str = "http://localhost:5001"
-    # whether to run on alchemy
-    alchemy: bool = False
-    # url for retrieving the contract artifacts
-    artifacts_url: str = "http://localhost:80"
-    # whether to run on devnet
-    devnet: bool = True
+    # hostname for container URLs
+    hostname: str = "http://localhost"
```

Comment:
This is making the assumption that these services are all on the same machine. Is this always the case?